activity.py

```diff
@@ -19,12 +19,13 @@
 sys.path.insert(0, "lib")
 import gi
 gi.require_version('Gtk', '3.0')
-from gi.repository import Gtk
+from gi.repository import Gtk, Gdk
```

Comment:
I didnot get a PYGIWarning when I tested, nevermind I will check if it's required and add it!

Reply:
Ok

Reply:
PyGIWarning for Gdk module tends to occur when odd versions of GTK are used by the tester.

By the way, the multiple imports on same line is unusual, and I thought flake8 would have said something about it.

To avoid unnecessary flake8 changes during a pull request, my technique is to checkout _master_, run flake8 and save the output, then checkout my branch, run flake8 again to a different file, and compare with diff or meld.

On large rewrites, it can be helpful to do the flake8 fixes _first_ before anything else.

Reply:
As suggested[ here](https://www.flake8rules.com/rules/E401.html), Imports from different modules should occur on different lines, It will give E401-Multiple imports on one line.
I would suggest doing flake8 fixes first.

Reply:
Done in 49afaa6 for the code patches I changed.